File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Get height from user
    int height;
    do
    {
        height = get_int("Enter pyramid height: ");    
    }
    while (height < 1 || height > 8);
    
    //Print pyramid
    for (int i = height - 1; i >= 0; i--)
    {
        //print reverse pyramid
        for (int j = 0; j < height; j++)
        {
            if (j < i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        
        //print middle gap
        printf("  ");
        
        //print reg pyramid
        for (int j = height - 1; j >= 0; j--)
        {
            if (j >= i)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

i = 1
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(0, height):
    
    for j in range(height - 1, -1, -1):
        if j <= i:
            print("#", end="")
        else:
            print(" ", end="")
    print("  ", end="")
    
    for j in range(height):
        if j <= i:
            print("#", end="")
    print("\n")
    